### PR TITLE
breijo

### PR DESCRIPTION
**Pull request by josebreijo**
🎉

### CODE REVIEW

File: src/components/Navigation/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import {
+  NavigationWrapperStyled,
+  LinkWrapperStyled
+} from "./Navigation.styled";
+
+const Navigation = () => (
+  <NavigationWrapperStyled>
+    <LinkWrapperStyled>
+      <NavLink to="/">home</NavLink>
+      <NavLink to="/favs">favs</NavLink>
```
Comment (jfer-rubio) [inline src/components/Navigation/Navigation.js]: missing activeClassName

File: src/App.js [hunk 2]
```diff
@@ -1,21 +1,40 @@
...
+    <Router>
+      <AppWrapperStyled>
+        <Navigation />
+        <Switch>
+          <Route exact path="/" component={Home} />
+          <Route exact path="/favs" component={Favs} />
+          <Route exact path="/details/:id" component={Details} />
+          <Route exact path="/search/:query" component={SearchResults} />
+          <Route component={NotFound} />
+          <Route />
```
Comment (jfer-rubio) [inline src/App.js]: extra route

File: src/App.js [hunk 3]
```diff
@@ -1,21 +1,40 @@
...
+  <Provider store={store}>
+    <Router>
+      <AppWrapperStyled>
+        <Navigation />
+        <Switch>
+          <Route exact path="/" component={Home} />
+          <Route exact path="/favs" component={Favs} />
+          <Route exact path="/details/:id" component={Details} />
+          <Route exact path="/search/:query" component={SearchResults} />
+          <Route component={NotFound} />
```
Comment (jfer-rubio) [inline src/App.js]: nice 👍

File: src/App.styled.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import styled, { injectGlobal } from "styled-components";
+
+injectGlobal`
```
Comment (Robfz) [inline src/App.styled.js]: 💯

**Pull request closed without merging**